.rubocop.yml

```diff
@@ -1,38 +1,38 @@
+# TODO: remove this file by either moving cops here or fixing code
+inherit_from:
+  - .rubocop_todo.yml
```

Comment:
Yes, the todo is needed.

Running rubocop via the foreman directory yield different results than running it within the plugin dir. Running rubocop without it in the plugin directory should give about 150 or more offences.

It is generated via the `--auto-gen-config` flag (running in the plugin dir) and i did not want to have them autocorrected, since it will mean that a number of files are touched that would cause a few conflicts with pending PRs.